### PR TITLE
Add option to switch off notifications

### PR DESCRIPTION
**Pull request by dybi**
resolves: #478

### CODE REVIEW

File: managers/models.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ class Project(models.Model):
     suspended = models.BooleanField(default=False)
     managers = models.ManyToManyField(CustomUser, related_name="manager_projects")
     members = models.ManyToManyField(CustomUser, related_name="projects")
+    notifications_on = models.BooleanField(default=True)
```
Comment (kbeker) [inline managers/models.py]: Hmm, maybe `notification_enabled`? :)

File: users/tests/test_views.py [hunk 2]
```diff
@@ -324,3 +324,15 @@ def test_manager_should_only_get_notifications_about_employees_from_his_projects
...
+        self.project.notifications_on = False
+        self.project.save()
+
+        with freeze_time("2019-07-08"):
+            ReportFactory(author=self.employee, project=self.project, date="2019-07-08")
+
+        self.client.force_login(self.manager)
+        with freeze_time("2019-07-15"):
+            response = self.client.get(self.url)
+        self._check_response(response, 200, ["No new notifications about employees in your projects"])
```
Comment (kbeker) [inline users/tests/test_views.py]: Please use `UserNotificationsText.NO_MORE_NOTIFICATIONS.value`.

File: managers/models.py (hunk 1 above)
Comment (rwrzesien) [reply managers/models.py]: Or `is_notification_enabled`.

**Comment by rwrzesien** [comment]
I think it is actually missing excluding those disabled projects from queryset when calculating. But I am not sure how it is supposed to work, as the calculation is done per user, not per project. So in example:
There is an user who is added to project A, B and C.
He was not reporting for 6 days in any project.
You disable notifications for project C.
What does it change? :)

**Comment by kbeker** [comment]
I might be wrong but notifications might need more features in feature #MakeNotificationGreatAgain

**Pull request merged**